go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -1263,6 +1274,40 @@ func (tsv *TabletServer) execDML(ctx context.Context, target *querypb.Target, qu
 	return int64(qr.RowsAffected), nil
 }
 
+// VStream streams VReplication events.
+func (tsv *TabletServer) VStream(ctx context.Context, target *querypb.Target, startPos mysql.Position, filter *binlogdatapb.Filter, send func([]*binlogdatapb.VEvent) error) error {
+	// This code is partially duplicated from execRequest. This is because
```

Comment:
could you add more context here? I was trying where is the copy coming from in `func (tsv *TabletServer) execRequest(`, but it was not obvious.  Did you mean startRequest?

Reply:
Yeah. It's startRequest.